snaps/tutorials/gas-estimation.md

```diff
@@ -12,31 +12,38 @@ information in a confirmation dialog.
 ## Prerequisites
 
 - [Snaps installed](../get-started/install-snaps.md)
-
 - A text editor (for example, [VS Code](https://code.visualstudio.com/))
-
 - [Yarn](https://yarnpkg.com/) version 3
 
 ## Steps
 
 ### 1. Set up the project
```

Comment:
Can you make similar changes to step 1 of the transaction insights snap tutorial?

Reply:
I will take this up in a separate PR. You can approve this and I will merge it.

Reply:
This: https://github.com/MetaMask/metamask-docs/pull/835